manager/manager/typeresolver.go

```diff
@@ -92,18 +93,27 @@ func (tr *typeResolver) ResolveTypes(config *common.Configuration, imports []*co
 	}
 
 	fetched := map[string][]*common.ImportFile{}
-	toFetch := make([]string, 0, tr.maxUrls)
+	// TODO(vaikas): Need to account for multiple URLs being fetched for a given type.
+	toFetch := make([]*fetchUnit, 0, tr.maxUrls)
```

Comment:
I wanted to chat about that, but for now I'd like to leave the TODO in place instead of ditch it. Many changes already, so I'd rather not do that here. Ok?